libsodium-sys/.gitignore

```diff
@@ -0,0 +1,30 @@
+#### IDEs ####
```

Comment:
Do we need this file?

Reply:
Its not strictly needed, but its practical when you are `git add .` from a subdirectory as opposed to the crate's root. I can remove it if you prefer.

Reply:
Its not an issue that you encounter if you use an IDE for instance.